ckanext/unaids/i18n/fr/LC_MESSAGES/ckanext-unaids.po

```diff
@@ -1039,30 +1040,30 @@ msgstr "Mes organisations"
 
 #: ckanext/unaids/theme/templates/user/edit_user_form.html:7
 msgid "Change details"
-msgstr ""
+msgstr "Changer les détails"
 
 #: ckanext/unaids/theme/templates/user/edit_user_form.html:13
 msgid "Full name"
-msgstr ""
+msgstr "Nom complet"
 
 #: ckanext/unaids/theme/templates/user/edit_user_form.html:13
 msgid "eg. Joe Bloggs"
-msgstr ""
+msgstr "exemple: Joe Bloggs"
 
 #: ckanext/unaids/theme/templates/user/edit_user_form.html:15
 msgid "eg. joe@example.com"
-msgstr ""
+msgstr "exemple: joe@example.com"
 
 #: ckanext/unaids/theme/templates/user/edit_user_form.html:17
 #: ckanext/unaids/theme/templates/user/new_user_form.html:18
 msgid "Job Title"
-msgstr ""
+msgstr "Titre du poste"
 
 #: ckanext/unaids/theme/templates/user/edit_user_form.html:17
 #: ckanext/unaids/theme/templates/user/new_user_form.html:18
 #, fuzzy
```

Comment:
When you're happy with a translation can you remove all the `#, fuzzy` tags please?

Reply:
Solved with last commit